pkg/defaultRules/defaultRules.yaml

```diff
@@ -1869,3 +1869,96 @@ rules:
         $ref: "#"
       items:
         $ref: "#"
+  - id: 61
+    name: "Prevent setting 'Unconfined' as seccomp profile"
```

Comment:
The name should be about what this rule does, and not how it does it. Something like:
"Prevent containers from having unnecessary system call privileges"
what do you think?